kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -454,6 +455,7 @@ public class KafkaRestConfig extends RestConfig {
   private static final String RATE_LIMIT_COSTS_DEFAULT = "";
 
   private static final ConfigDef config;
+  private Metrics metrics;
```

Comment:
I think this needs to be `volatile` as the metrics seem to be written and read in a concurrent fashion (written in the app initialization, read whenever we initialize produce metrics).